src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/task.py

```diff
@@ -351,6 +365,11 @@ def acr_task_update(cmd,  # pylint: disable=too-many-locals
         trigger=TriggerUpdateParameters(
             source_triggers=source_trigger_update_params,
             base_image_trigger=base_image_trigger_update_params
+        ),
+        credentials=get_credentials(
+            cmd,
+            auth_mode=auth_mode,
```

Comment:
In the update path, if user didn't specify the auth mode, it means the existing auth mode should be retained.

Reply:
Will the above get_credentials reset to default when auth mode is not given?

Reply:
Is there a gesture to remove an existing credential in the update path. The RP model for credentials is a dictionary and supports JSON merge syntax where if you send a null value for the key, that credential will be removed

Reply:
yes it will work

Reply:
@shahzzzam, I see @mnltejaswini comments said you need to pass null value to remove the existing credentials. I think it's different from passing empty username and password. Did you check it?  

```
            username = cred_split[1].strip()
            password = cred_split[2].strip()
            custom_registries[login_server] = CustomRegistryCredentials(
                user_name=SecretObject(
                    type=SecretObjectType.opaque,
                    value=username
                ),
                password=SecretObject(
                    type=SecretObjectType.opaque,
                    value=password
                )
            )
```